experimental/packages/otlp-grpc-exporter-base/package.json

```diff
@@ -49,6 +49,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.23.6",
+    "@babel/preset-env": "7.22.20",
```

Comment:
Hmm, do we need this here? :thinking: 
The gRPC exporter only supports Node.js so we never use it right? :thinking: We may also be able to drop `@babel/core`.

Reply:
Well, yeah, thank you for spotting this. Unnecessary test dependencies removed.